packages/network/src/api/inspection/mappers.ts

```diff
@@ -568,3 +624,66 @@ export function mapApiInspectionPost(options: CreateInspectionOptions): ApiInspe
     },
   };
 }
+
+interface PaginationRequestParams {
+  /**
+   * The number of inspections fetched.
+   *
+   * @default 100
+   */
+  limit?: number;
+  /**
+   * The date (in ISO 8601 format) or the inspection ID to filter inspections that occurred before this date or ID.
+   */
+  before?: string;
+
+  /**
+   * The date (in ISO 8601 format) or the inspection ID. to filter inspections that occurred after this date.
+   */
+  after?: string;
+  /**
+   * The order of the pagination.
+   *
+   * @default SortOrder.DESC
+   */
+  paginationOrder?: SortOrder;
```

Comment:
For me, the sort order is not part of the pagination. The sort is its own thing. There are 3 elements when displaying a paginated list of things :
- Filters (which you included in the options)
- Pagination (specifying the cursor and limit and just that)
- Sort (which contains the sort by property and the sort direction)

All of these items should be at the root of the request options interface